### ISSUE TITLE
Not working on an ultrawide monitor

### ISSUE THREAD

**Issue by alonso-m** [b12f/wlstreamer#1]
The video output remains on the same focus regardless of the app displaying the message "Switched focus". I'm on a 5120x1440 resolution.

Any info that could help troubleshooting the case let me know.

**Comment by b12f** [comment]
Do you have multiple monitors with different resolutions or just that one resolution?

**Comment by b12f** [comment]
Also, which application did you test this with? Can you try with `vlc`?

**Comment by alonso-m** [comment]
Just one monitor. What I noticed is that it records the same part of the screen as `wf-recorder` when I don't specify a geometry. That's always the center part of the screen. It happens with any apps, terminal, browser, slack, etc.

**Comment by b12f** [comment]
Ah I understand. Currently wlstreamer only switches _screens_ being recorded, not applications. What you can do however if you want to hide certain applications is to put them on different workspaces, and then use `--not-ws $ws-num` to hide that workspace.

It is probably somehow feasible to do application switching too, but it is outside of my personal usecase, so I'll accept a PR but not do any work on it. One issue I might see arising is the fast difference in resolutions/aspect ratios between applications, resulting in a video feed that is very hard to follow for viewers.

As a workaround, I think some ultrawide monitors allow you to input via two different cables, "faking" a double screen setup. This way sway would report two monitors, allowing you to show/hide either one.

**Comment by alonso-m** [comment]
Thanks for the info! It's clear now.
